types/bero/bero-tests.ts

```diff
@@ -7,6 +7,10 @@ const block1 = bem('block', ['modifier1', undefined, 'modifier2']);
 const block2 = bem('block');
 const classesElement2 = join(block2('element', ['modifier']), 'extra-element-class');
 
+const classesElement3 = join(block2('element', ['modifier']), undefined);
+
+const classesElement4 = join(block2('element', undefined), undefined);
```

Comment:
```suggestion
const classesElement4 = join(block2('element', undefined), undefined);
const classesElement4 = join(block2('element', undefined));
const classesElement4 = join(block2('element'));
```

Reply:
Fixed also this @peterblazejewicz thanks